Student/Student/Submissions/SubmissionRubric/RubricPresenter.swift

```diff
@@ -101,7 +102,7 @@ class RubricPresenter {
                 comments = assessment.comments
             }
 
-            let m = RubricViewModel(title: r.desc, selectedDesc: description, selectedIndex: selectedIndex, ratings: allRatings, comment: comments)
+            let m = RubricViewModel(title: r.desc, longDescription: r.longDesc, selectedDesc: description, selectedIndex: selectedIndex, ratings: allRatings, comment: comments)
```

Comment:
I'm not sure, but if it's a custom assessment, would it have a long description?

Reply:
I played around with this a bit and found that the long description will always be a string. If there is no long description it is just empty.

Reply:
👍